projects/1.16/assets/tropicraft/tropicraft/lang/zh_cn.json

```
@@ -0,0 +1,496 @@
+  "item.tropicraft.bamboo_mug": "竹杯",
+  "item.tropicraft.bamboo_spear": "竹矛",
+  "item.tropicraft.bamboo_stick": "竹棍",
+  "item.tropicraft.black_beach_float": "黑色沙滩浮排",
+  "item.tropicraft.black_chair": "黑色椅子",
+  "item.tropicraft.black_coffee": "黑咖啡",
+  "item.tropicraft.black_pearl": "黑珍珠",
+  "item.tropicraft.black_umbrella": "黑色伞",
+  "item.tropicraft.blow_gun": "吹气枪",
+  "item.tropicraft.blue_beach_float": "蓝色沙滩浮排",
+  "item.tropicraft.blue_chair": "蓝色椅子",
+  "item.tropicraft.blue_umbrella": "蓝色伞",
+  "item.tropicraft.brown_basilisk_lizard_spawn_egg": "棕色蜥蜴刷怪蛋",
+  "item.tropicraft.brown_beach_float": "棕色沙滩浮排",
+  "item.tropicraft.brown_chair": "棕色椅子",
+  "item.tropicraft.brown_umbrella": "棕色伞",
+  "item.tropicraft.caipirinha": "卡琵利亚",
+  "item.tropicraft.cocktail": "鸡尾酒",
+  "item.tropicraft.coconut_chunk": "椰子片",
+  "item.tropicraft.coconut_water": "椰子汁",
+  "item.tropicraft.coffee_berry": "咖啡果",
+  "item.tropicraft.cooked_fish": "熟鱼",
+  "item.tropicraft.cooked_frog_leg": "熟青蛙腿",
+  "item.tropicraft.cooked_ray": "熟鳐鱼",
+  "item.tropicraft.cowktail_spawn_egg": "牛爱花刷怪蛋",
+  "item.tropicraft.cubera_spawn_egg": "巴西笛鲷刷怪蛋",
+  "item.tropicraft.cyan_beach_float": "青色沙滩浮排",
+  "item.tropicraft.cyan_chair": "青色椅子",
+  "item.tropicraft.cyan_umbrella": "青色伞",
+  "item.tropicraft.dagger": "匕首",
+  "item.tropicraft.dolphin_spawn_egg": "海豚刷怪蛋",
+  "item.tropicraft.eagle_ray_spawn_egg": "鹰鳐刷怪蛋",
+  "item.tropicraft.eih_spawn_egg": "复活节岛石像刷怪蛋",
+  "item.tropicraft.eudialyte_axe": "异性石斧",
+  "item.tropicraft.eudialyte_gem": "异性石宝石",
+  "item.tropicraft.eudialyte_hoe": "异性石锄",
+  "item.tropicraft.eudialyte_pickaxe": "异性石镐",
+  "item.tropicraft.eudialyte_shovel": "异性石铲",
+  "item.tropicraft.eudialyte_sword": "异性石剑",
+  "item.tropicraft.exploding_coconut": "椰子炸弹",
+  "item.tropicraft.failgull_spawn_egg": "海鸟刷怪蛋",
+  "item.tropicraft.fiddler_crab_spawn_egg": "招潮蟹刷怪蛋",
+  "item.tropicraft.fire_boots": "防火靴子",
+  "item.tropicraft.fire_chestplate": "防火胸甲",
+  "item.tropicraft.fire_helmet": "防火头盔",
+  "item.tropicraft.fire_leggings": "防火护腿",
+  "item.tropicraft.fishing_net": "渔网",
+  "item.tropicraft.fresh_marlin": "生马林鱼",
+  "item.tropicraft.frog_leg": "青蛙腿",
+  "item.tropicraft.frox_conch": "Frox海螺",
+  "item.tropicraft.grapefruit": "西柚",
+  "item.tropicraft.gray_beach_float": "灰色沙滩浮排",
+  "item.tropicraft.gray_chair": "灰色椅子",
+  "item.tropicraft.gray_umbrella": "灰色伞",
+  "item.tropicraft.green_basilisk_lizard_spawn_egg": "绿色蜥刷怪蛋",
+  "item.tropicraft.green_beach_float": "绿色沙滩浮排",
+  "item.tropicraft.green_chair": "绿色椅子",
+  "item.tropicraft.green_umbrella": "绿色伞",
+  "item.tropicraft.hammerhead_spawn_egg": "锤头鲨刷怪蛋",
+  "item.tropicraft.hummingbird_spawn_egg": "蜂鸟刷怪蛋",
+  "item.tropicraft.iguana_leather": "鬣蜥皮",
+  "item.tropicraft.iguana_spawn_egg": "鬣蜥刷怪蛋",
+  "item.tropicraft.jaguar_spawn_egg": "美洲豹刷怪蛋",
+  "item.tropicraft.koa_spawn_egg": "原始人刷怪蛋",
+  "item.tropicraft.lemon": "柠檬",
+  "item.tropicraft.lemonade": "柠檬汁",
```

```suggestion
  "item.tropicraft.lemonade": "柠檬水",
```
不知道是不是这样，但看名称`柠檬水`准确一些，此处修改则下方`青柠汁`也需修改，`Orangeade`比较麻烦

---

这些饮料都是在该mod内一个类似榨汁机的功能方块（饮料搅拌机）上制作的，
参考蜜雪冰城4块钱一杯的柠檬水，因是往水里加柠檬片来泡的，主要是水故称 `柠檬水`，此饮料游戏内制作过程中，机器有较为明显的晃动动作，猜测更像是在榨汁，故个人认为采取 `柠檬汁` 译法